doc/classes/AnimatedSprite2D.xml

```diff
@@ -101,6 +106,11 @@
 		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
 			The texture's drawing offset.
 		</member>
+		<member name="pivot_mode" type="int" setter="set_pivot_mode" getter="get_pivot_mode" enum="Texture2D.Pivot" default="3">
+			The sprite's pivot point, relative to the texture.
+
+			[b]Note:[/b] For games with a pixel art aesthetic, textures may appear deformed when centered. This is caused by their position being between pixels. To prevent this, set this property to [code]false[/code], or consider enabling [member ProjectSettings.rendering/2d/snap/snap_2d_vertices_to_pixel] and [member ProjectSettings.rendering/2d/snap/snap_2d_transforms_to_pixel].
```

Comment:
This is copied straight from `centered` which makes no sense here.

Reply:
Shall I put the note in the constant description instead ? 
I must say that, currently, the way the property's hover popup is displayed make it already hard to access all the possible values (the popup shows a scrollbar, but trying to mouse wheel over it doesn't work because it disappears as soon as the mouse leaves the property).

Reply:
Ideally, yes, unless you also specify the constant itself (`[constant Texture2D.PIVOT_CENTER]`) instead of saying "centered".

```suggestion
			[b]Note:[/b] For games with a pixel art aesthetic, textures may appear deformed when this property is set to [constant Texture2D.PIVOT_CENTER]. This is caused by their position being between pixels. To prevent this, set this property to [code]false[/code], or consider enabling [member ProjectSettings.rendering/2d/snap/snap_2d_vertices_to_pixel] and [member ProjectSettings.rendering/2d/snap/snap_2d_transforms_to_pixel].
```

> I must say that, currently, the way the property's hover popup is displayed make it already hard to access all the possible values

I can relate. A majority of constant descriptions are written short, but that's not necessarily because of this reason. Assume that the tooltip is working fine and that you're reading this from the built-in docs. Improvements to the tooltips are not related.

Reply:
I ended up putting the notes in the constants, given several modes can produce positions in-between pixels it makes more sense to have the ability to tell which mode may trigger a deformed texture.